docs/cloud-native-security/index.rst

```diff
@@ -4,14 +4,10 @@
 Cloud Native Security
 =====================
 
-AWS is constantly expanding its portfolio of native security products. Thanks
-to its more holistic access to backend data and resources, Amazon is able to
-offer security features that 3rd party tools are unable to provide. Rackspace
-is expanding its managed security services to provide support for native
-AWS security products.
+AWS is constantly expanding its portfolio of native security products. Thanks to its more holistic access to backend data and resources, Amazon is able to offer security features that 3rd party tools are unable to provide. Rackspace is expanding its managed security services to provide SOC-as-a-Service for native AWS security products.
```

Comment:
... Amazon can offer security features that third-party tools cannot provide. ...

Reply:
updated